modules/platforms/cpp/thin-client/include/ignite/thin/ignite_client_configuration.h

```diff
@@ -230,6 +231,29 @@ namespace ignite
                 return partitionAwareness;
             }
 
+            /**
+             * Get connection limit.
```

Comment:
Let's explain in more detail, something like `with Partition Awareness enabled, thin client establishes a connection to every server node in the cluster. Use this setting to limit the number of active connections. This reduces the resource usage, but can have a negative effect on cache operation performance`.

Reply:
Actually, it does so with partition awareness disabled as well. It was done for fast failover and load balancing. But I've got your point, will add a more detailed description.